torch/optim/lr_scheduler.py

```diff
@@ -252,17 +230,12 @@ class MultiStepLR(_LRScheduler):
     """
 
     def __init__(self, optimizer, milestones, gamma=0.1, last_epoch=-1):
-        self.milestones = Counter(milestones)
+        self.milestones = milestones
+        self._milestones_counter = Counter(milestones)
```

Comment:
It's probably worth a comment here that this Counter is here specifically for a subclass instance (which lives in another file).

Actually, this is one of the reasons why I preferred the version of the code where the closed-form and recursive formulas all lived in one class, it means that all of the formulas for a given scheduler were in one place and you wouldn't easily forget to update one if you updated the other (and similarly, if you added a new LR scheduler, you would remember to add another LR scheduler.) Maybe "chainability" should still be toggled as a keyword argument, rather than blasting out an entirely separate module hierarchy?